{
    "title" : "503 An error occurred while making the HTTP request to https:__yourCESMasterServer:52810_7.0_CoveoSearchService",
    "uri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA1320000000428CAA/language:en_US",
    "printableUri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA1320000000428CAA/language:en_US",
    "clickUri" : "https://na61.salesforce.com/kA1320000000428",
    "uniqueId" : "42.13819$http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA1320000000428CAA/language:en_US",
    "excerpt" : "This could be due to the fact that the server certificate is not configured properly with HTTP.SYS in the HTTPS case. ... This could also be caused by a mismatch of the security binding between the...",
    "firstSentences" : null,
    "summary" : null,
    "flags" : "HasHtmlVersion",
    "hasHtmlVersion" : true,
    "hasMobileHtmlVersion" : false,
    "score" : 1397,
    "percentScore" : 84.103615,
    "rankingInfo" : null,
    "rating" : 3.0,
    "isTopResult" : false,
    "isRecommendation" : false,
    "titleHighlights" : [ ],
    "firstSentencesHighlights" : [ ],
    "excerptHighlights" : [ ],
    "printableUriHighlights" : [ ],
    "summaryHighlights" : [ ],
    "parentResult" : null,
    "childResults" : [ ],
    "totalNumberOfChildResults" : 0,
    "raw" : {
      "systitle" : "503 An error occurred while making the HTTP request to https:__yourCESMasterServer:52810_7.0_CoveoSearchService",
      "sysurihash" : "ðKON0AJ9gyg9utðx",
      "urihash" : "ðKON0AJ9gyg9utðx",
      "parents" : "<?xml version=\"1.0\" encoding=\"utf-16\"?><parents><parent name=\"Organization\" uri=\"https://na61.salesforce.com/home/home.jsp\" /><parent name=\"KB_Article__ka\" uri=\"http://www.salesforce.com/org:organization/articletype:KB_Article\" /><parent name=\"503 An error occurred while making the HTTP request to https:__yourCESMasterServer:52810_7.0_CoveoSearchService\" uri=\"https://na61.salesforce.com/kA1320000000428\" /></parents>",
      "sfdcarticletype" : "Problem/Fix",
      "sfresolutionc" : "Since it&#39;s aService that&#39;s using port 52810, you need to:<br><br>1 - Stop aService<br>2 - Wait approximately 2min for the port to be redistributed<br>3 - Restart CES7 Service from Windows Services<br>4 - Restart aService<br><br>This way, CES7Service will reappropriate port 52810",
      "sysuri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA1320000000428CAA/language:en_US",
      "sysprintableuri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA1320000000428CAA/language:en_US",
      "sflastpublisheddate" : 1481913895000,
      "systransactionid" : 161139,
      "sfversionc" : "7",
      "sfurlname" : "1053",
      "sfsystemmodstamp" : 1517572013000,
      "sfisvisibleinapp" : "true",
      "sfcoveoversionc" : "Coveo for Sitecore 3.0",
      "sysconcepts" : "server ; port ; window ; command ; Coveo infrastructure ; mismatch of the security ; 2min ; Wait ; repository ; Administrator",
      "sfkbversionnumber" : "3",
      "concepts" : "server ; port ; window ; command ; Coveo infrastructure ; mismatch of the security ; 2min ; Wait ; repository ; Administrator",
      "printableuri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA1320000000428CAA/language:en_US",
      "documenttype" : "article",
      "sfkbarticlekavid" : "ka132000000LJxyAAG",
      "sysindexeddate" : 1518187568000,
      "sfknowledgearticleid" : "kA1320000000428CAA",
      "sfcreatedbyid" : "00560000003q7JgAAI",
      "permanentid" : "c0ab3e69ff69d2d0ca512c74d530b69dcc58372df93595a71ae55e089664",
      "syslanguage" : [ "English" ],
      "transactionid" : 161139,
      "sfisvisibleinprm" : "true",
      "title" : "503 An error occurred while making the HTTP request to https:__yourCESMasterServer:52810_7.0_CoveoSearchService",
      "sfversionnumber" : 3,
      "sfarticlenumber" : "000001053",
      "date" : 1517572013000,
      "objecttype" : "KB Article",
      "allmetadatavalues" : "{\"kav_lastmodifiedby.attribute.type\": [\"User\"], \"coveo_short_id\": [\"kA1320000000428\"], \"sflastmodifiedbyname\": [\"Frederic Cote\"], \"kav_coveo_version__c__html_stripped\": [\"Coveo for Sitecore 3.0\"], \"fileextension\": [\"\"], \"sfdcexpandedproduct\": [\"All;All|Sitecore\"], \"kav_createdby.attribute.url\": [\"/services/data/v34.0/sobjects/User/00560000003q7JgAAI\"], \"sflanguage\": [\"en_US\"], \"kav_createdby.attribute.type\": [\"User\"], \"kav_createdby.id\": [\"00560000003q7JgAAI\"], \"kav_language\": [\"en_US\"], \"sourcetype\": [\"Salesforce\"], \"title\": [\"503 An error occurred while making the HTTP request to https:__yourCESMasterServer:52810_7.0_CoveoSearchService\"], \"source\": [\"Salesforce Public Items\"], \"kav_owner.attribute.url\": [\"/services/data/v34.0/sobjects/User/00560000003q7JgAAI\"], \"parents\": [\"<?xml version=\\\"1.0\\\" encoding=\\\"utf-16\\\"?><parents><parent name=\\\"Organization\\\" uri=\\\"https://na61.salesforce.com/home/home.jsp\\\" /><parent name=\\\"KB_Article__ka\\\" uri=\\\"http://www.salesforce.com/org:organization/articletype:KB_Article\\\" /><parent name=\\\"503 An error occurred while making the HTTP request to https:__yourCESMasterServer:52810_7.0_CoveoSearchService\\\" uri=\\\"https://na61.salesforce.com/kA1320000000428\\\" /></parents>\"], \"sfisvisibleinprm\": [true], \"kav_title\": [\"503 An error occurred while making the HTTP request to https://yourCESMasterServer:52810/7.0/CoveoSearchService\"], \"kav_isvisibleincsp\": [true], \"hashtmlversion\": [true], \"sfarticletypec\": [\"Error Message[Deprecated]\"], \"kav_version__c\": [\"7\"], \"kav_versionnumber\": [3], \"sfcreatedbyname\": [\"Frederic Cote\"], \"sflastpublisheddate\": [1481913895], \"objecttypelabel\": [\"KB Article\"], \"kav_cause__c__html_stripped\": [\"If the port is open, then it means that the port 52810 was attributed to another service. \\n \\n Port 52810 is essential to the Coveo infrastructure. \\n \\n To validate this assumption, open a command line Window as an Administrator and run: \\n \\n netstat -baon > output.txt \\n \\n This will generate a output.txt file in the current repository of your command line window. \\n \\n Inside that file, search for 52810. You will find a line like this: \\n \\n TCP 0.0.0.0:52810 0.0.0.0:0 LISTENING 1234\\u00a0 \\n [aService.exe] \\n \\n *1234 is the PID of aService.exe \\n \\u00a0\"], \"originalhtmlcharset\": [1200], \"coveo_connector_hasbinarydata\": [false], \"kav_lastpublisheddate\": [1481913895], \"sfsystemmodstamp\": [1517572013], \"ka_attribute.type\": [\"KB_Article__ka\"], \"sfdcproduct\": [\"Sitecore\"], \"kav_resolution__c__html_stripped\": [\"Since it's aService that's using port 52810, you need to: \\n \\n 1 - Stop aService \\n 2 - Wait approximately 2min for the port to be redistributed \\n 3 - Restart CES7 Service from Windows Services \\n 4 - Restart aService \\n \\n This way, CES7Service will reappropriate port 52810\"], \"clickableuri\": [\"https://na61.salesforce.com/kA1320000000428\"], \"kav_dc_expanded_product\": [\"All;All|Sitecore\"], \"sftitle\": [\"503 An error occurred while making the HTTP request to https://yourCESMasterServer:52810/7.0/CoveoSearchService\"], \"kav_owner.id\": [\"00560000003q7JgAAI\"], \"sfbuildc\": [\"All\"], \"sfisvisibleinapp\": [true], \"ka_attribute.url\": [\"/services/data/v34.0/sobjects/KB_Article__ka/kA1320000000428CAA\"], \"sfdcarticletype\": [\"Problem/Fix\"], \"kav_cause__c\": [\"If the port is open, then it means that the port 52810 was attributed to another service.<br><br>Port 52810 is essential to the Coveo infrastructure.<br><br>To validate this assumption, open a command line Window as an Administrator and run:<br><br>netstat -baon &gt; output.txt<br><br>This will generate a output.txt file in the current repository of your command line window.<br><br>Inside that file, search for 52810. You will find a line like this:<br><br>TCP 0.0.0.0:52810 0.0.0.0:0 LISTENING 1234\\u00a0<br>[aService.exe]<br><br>*1234 is the PID of aService.exe<br>\\u00a0\"], \"sfislatestversion\": [true], \"sfcausec\": [\"If the port is open, then it means that the port 52810 was attributed to another service.<br><br>Port 52810 is essential to the Coveo infrastructure.<br><br>To validate this assumption, open a command line Window as an Administrator and run:<br><br>netstat -baon &gt; output.txt<br><br>This will generate a output.txt file in the current repository of your command line window.<br><br>Inside that file, search for 52810. You will find a line like this:<br><br>TCP 0.0.0.0:52810 0.0.0.0:0 LISTENING 1234\\u00a0<br>[aService.exe]<br><br>*1234 is the PID of aService.exe<br>\\u00a0\"], \"contenttype\": [\"binarydata\"], \"sfurlname\": [\"1053\"], \"ka_id\": [\"kA1320000000428CAA\"], \"kav_datacategoryselections.datacategoryname\": [\"Sitecore;Configuration\"], \"coveo_url\": [\"https://na61.salesforce.com/kA1320000000428\"], \"kav_urlname\": [\"1053\"], \"kav_articletype\": [\"KB_Article__kav\"], \"foldingcollection\": [\"bdbc5dd88ef954551425c1e7ce7bab9d2b8a105f30d7b6a09d79e89a67e3\"], \"body\": [\"<html>   <body>     <div style=\\\"background-color: #fff;border: 2px solid #BBC5CC;border-radius: 4px;float: left;font-family: lato;margin-left: 15px;margin-right: 15px;margin-bottom: 4%;padding: 10px;width: 60%;\\\">       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Description         </div>       </div>       <div style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;\\\"> <b>Symptoms</b><br><br>1 - On you Coveo for Sitecore diagnostic page you see:<br><br><i>503 An error occurred while making the HTTP request to https://yourCESMasterServer:52810/7.0/CoveoSearchService. This could be due to the fact that the server certificate is not configured properly with HTTP.SYS in the HTTPS case. This could also be caused by a mismatch of the security binding between the client and the server</i><br><br>2 - When calling restApiServer:8080/rest/search [Default port is 8080, can be changed in config.yml]<br><br>&quot;message&quot; : &quot;Server is unavailable: (https://<span style=\\\"\\\">yourCESMasterServer</span>:52810/7.0/CoveoSearchService,java.net.ConnectException: https://yourCESMasterServer:52810/7.0/CoveoSearchService)&quot;<br><br>3 - On your CESMasterServer, you get an &quot;Timeout&quot; message if you&#39;re trying to connect to:<br><br><i>https://localhost:52810/7.0/CoveoSearchService?wsdl</i><br><br>4 -\\u00a0<span style=\\\"\\\">On your CESMasterServer, this command indicates that port 52810 is open<br><br>telnet localhost 52810</span></div>       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Environment         </div>       </div>       <br/>       <table>         <tr>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Product</td>           <td>Coveo Enterprise Search</td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Sitecore Version</td>           <td></td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Connector</td>           <td></td>         </tr>         <tr>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Published</td>           <td>1481913895</td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Version</td>           <td>7</td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Build</td>           <td>All</td>         </tr>       </table>       <br/>       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Cause         </div>       </div>       <div style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;\\\"> If the port is open, then it means that the port 52810 was attributed to another service.<br><br>Port 52810 is essential to the Coveo infrastructure.<br><br>To validate this assumption, open a command line Window as an Administrator and run:<br><br>netstat -baon &gt; output.txt<br><br>This will generate a output.txt file in the current repository of your command line window.<br><br>Inside that file, search for 52810. You will find a line like this:<br><br>TCP 0.0.0.0:52810 0.0.0.0:0 LISTENING 1234\\u00a0<br>[aService.exe]<br><br>*1234 is the PID of aService.exe<br>\\u00a0</div>       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Resolution         </div>       </div>       <div style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;\\\"> Since it&#39;s aService that&#39;s using port 52810, you need to:<br><br>1 - Stop aService<br>2 - Wait approximately 2min for the port to be redistributed<br>3 - Restart CES7 Service from Windows Services<br>4 - Restart aService<br><br>This way, CES7Service will reappropriate port 52810</div>     </div>   </body> </html>\"], \"kav_product__c\": [\"Coveo Enterprise Search\"], \"kav_article_type__c\": [\"Error Message[Deprecated]\"], \"sfpublishstatus\": [\"Online\"], \"sfcreatedbyid\": [\"00560000003q7JgAAI\"], \"kav_description__c__html_stripped\": [\"Symptoms \\n \\n 1 - On you Coveo for Sitecore diagnostic page you see: \\n \\n 503 An error occurred while making the HTTP request to https://yourCESMasterServer:52810/7.0/CoveoSearchService. This could be due to the fact that the server certificate is not configured properly with HTTP.SYS in the HTTPS case. This could also be caused by a mismatch of the security binding between the client and the server \\n \\n 2 - When calling restApiServer:8080/rest/search [Default port is 8080, can be changed in config.yml] \\n \\n \\\"message\\\" : \\\"Server is unavailable: (https:// yourCESMasterServer :52810/7.0/CoveoSearchService,java.net.ConnectException: https://yourCESMasterServer:52810/7.0/CoveoSearchService)\\\" \\n \\n 3 - On your CESMasterServer, you get an \\\"Timeout\\\" message if you're trying to connect to: \\n \\n https://localhost:52810/7.0/CoveoSearchService?wsdl \\n \\n 4 -\\u00a0 On your CESMasterServer, this command indicates that port 52810 is open \\n \\n telnet localhost 52810\"], \"kav_islatestversion\": [true], \"kav_knowledgearticleid\": [\"kA1320000000428CAA\"], \"sfknowledgearticleid\": [\"kA1320000000428CAA\"], \"sfownername\": [\"Frederic Cote\"], \"kav_datacategoryselections.attribute.type\": [\"KB_Article__DataCategorySelection;KB_Article__DataCategorySelection\"], \"detectedtitle\": [\"Description\"], \"kav_createddate\": [1481913836], \"kav_build__c\": [\"All\"], \"language\": [\"English\"], \"coveo_foldingchild\": [\"d607de66575964e1dd0a57a1f08ee7cea1bf9b45ba146db613f4466f4e54\"], \"sfownerid\": [\"00560000003q7JgAAI\"], \"kav_createdby.coveo_record_size\": [2], \"kav_systemmodstamp\": [1517572013], \"objecttype\": [\"KB Article\"], \"connectortype\": [\"Salesforce2\"], \"kav_coveo_record_size\": [4], \"originaluri\": [\"http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA1320000000428CAA/language:en_US\"], \"sfisvisibleincsp\": [true], \"convertertype\": [\"TYPE_HTML\"], \"filetype\": [\"SalesforceItem\"], \"kav_owner.name\": [\"Frederic Cote\"], \"sfcoveoversionc\": [\"Coveo for Sitecore 3.0\"], \"sfdcexpandedarticletype\": [\"All;All|Problem/Fix\"], \"objecttypelabelplural\": [\"KB Articles\"], \"sfcurrencyisocode\": [\"CAD\"], \"sfisdeleted\": [false], \"extractedsize\": [3726], \"size\": [6119], \"kav_publishstatus\": [\"Online\"], \"generateexcerpt\": [true], \"sfdescriptionc\": [\"<b>Symptoms</b><br><br>1 - On you Coveo for Sitecore diagnostic page you see:<br><br><i>503 An error occurred while making the HTTP request to https://yourCESMasterServer:52810/7.0/CoveoSearchService. This could be due to the fact that the server certificate is not configured properly with HTTP.SYS in the HTTPS case. This could also be caused by a mismatch of the security binding between the client and the server</i><br><br>2 - When calling restApiServer:8080/rest/search [Default port is 8080, can be changed in config.yml]<br><br>&quot;message&quot; : &quot;Server is unavailable: (https://<span style=\\\"\\\">yourCESMasterServer</span>:52810/7.0/CoveoSearchService,java.net.ConnectException: https://yourCESMasterServer:52810/7.0/CoveoSearchService)&quot;<br><br>3 - On your CESMasterServer, you get an &quot;Timeout&quot; message if you&#39;re trying to connect to:<br><br><i>https://localhost:52810/7.0/CoveoSearchService?wsdl</i><br><br>4 -\\u00a0<span style=\\\"\\\">On your CESMasterServer, this command indicates that port 52810 is open<br><br>telnet localhost 52810</span>\"], \"sfarticletype\": [\"KB_Article__kav\"], \"kav_lastmodifieddate\": [1481913895], \"documenttype\": [\"article\"], \"sfkbarticlekavid\": [\"ka132000000LJxyAAG\"], \"kav_currencyisocode\": [\"CAD\"], \"sfproductc\": [\"Coveo Enterprise Search\"], \"foldingparent\": [\"d607de66575964e1dd0a57a1f08ee7cea1bf9b45ba146db613f4466f4e54\"], \"mappingtype\": [\"KB_Article\"], \"kav_isvisibleinpkb\": [true], \"kav_sourceid\": [\"5003200000vJYSiAAO\"], \"sfkbversionnumber\": [3], \"kav_isvisibleinapp\": [true], \"kav_resolution__c\": [\"Since it&#39;s aService that&#39;s using port 52810, you need to:<br><br>1 - Stop aService<br>2 - Wait approximately 2min for the port to be redistributed<br>3 - Restart CES7 Service from Windows Services<br>4 - Restart aService<br><br>This way, CES7Service will reappropriate port 52810\"], \"coveo_objecttype\": [\"KB_Article\"], \"printableuri\": [\"http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA1320000000428CAA/language:en_US\"], \"kav_datacategoryselections.attribute.url\": [\"/services/data/v34.0/sobjects/KB_Article__DataCategorySelection/02o32000000pukHAAQ;/services/data/v34.0/sobjects/KB_Article__DataCategorySelection/02o32000000pukDAAQ\"], \"kav_id\": [\"ka132000000LJxyAAG\"], \"sfresolutionc\": [\"Since it&#39;s aService that&#39;s using port 52810, you need to:<br><br>1 - Stop aService<br>2 - Wait approximately 2min for the port to be redistributed<br>3 - Restart CES7 Service from Windows Services<br>4 - Restart aService<br><br>This way, CES7Service will reappropriate port 52810\"], \"foldingchild\": [\"d607de66575964e1dd0a57a1f08ee7cea1bf9b45ba146db613f4466f4e54\"], \"sfcreateddate\": [1481913836], \"kav_owner.coveo_record_size\": [2], \"kav_coveo_effective_modstamp\": [1517572013], \"kav_summary__html_stripped\": [\"\"], \"kav_dc_article_type\": [\"Problem/Fix\"], \"sffirstpublisheddate\": [1453322867], \"kav_dc_product\": [\"Sitecore\"], \"ka_coveo_record_size\": [2], \"kav_dc_expanded_article_type\": [\"All;All|Problem/Fix\"], \"kav_datacategoryselections.coveo_record_size\": [\"2;2\"], \"concepts\": [\"server ; port ; window ; command ; Coveo infrastructure ; mismatch of the security ; 2min ; Wait ; repository ; Administrator\"], \"sfkbid\": [\"kA1320000000428CAA\"], \"kav_firstpublisheddate\": [1453322867], \"coveo_organization_id\": [\"00D3000000007r2\"], \"coveo_foldingcollection\": [\"bdbc5dd88ef954551425c1e7ce7bab9d2b8a105f30d7b6a09d79e89a67e3\"], \"sflastmodifieddate\": [1481913895], \"kav_coveo_version__c\": [\"Coveo for Sitecore 3.0\"], \"kav_vendor_version__c__html_stripped\": [\"\"], \"detectedlanguage\": [1], \"kav_isvisibleinprm\": [true], \"kav_owner.attribute.type\": [\"Name\"], \"kav_lastmodifiedby.attribute.url\": [\"/services/data/v34.0/sobjects/User/00560000003q7JgAAI\"], \"sfversionc\": [\"7\"], \"sitename\": [\"Community\"], \"kav_createdby.name\": [\"Frederic Cote\"], \"sfarticlenumber\": [\"000001053\"], \"kav_lastmodifiedby.coveo_record_size\": [2], \"sfisvisibleinpkb\": [true], \"kav_datacategoryselections.id\": [\"02o32000000pukHAAQ;02o32000000pukDAAQ\"], \"kav_description__c\": [\"<b>Symptoms</b><br><br>1 - On you Coveo for Sitecore diagnostic page you see:<br><br><i>503 An error occurred while making the HTTP request to https://yourCESMasterServer:52810/7.0/CoveoSearchService. This could be due to the fact that the server certificate is not configured properly with HTTP.SYS in the HTTPS case. This could also be caused by a mismatch of the security binding between the client and the server</i><br><br>2 - When calling restApiServer:8080/rest/search [Default port is 8080, can be changed in config.yml]<br><br>&quot;message&quot; : &quot;Server is unavailable: (https://<span style=\\\"\\\">yourCESMasterServer</span>:52810/7.0/CoveoSearchService,java.net.ConnectException: https://yourCESMasterServer:52810/7.0/CoveoSearchService)&quot;<br><br>3 - On your CESMasterServer, you get an &quot;Timeout&quot; message if you&#39;re trying to connect to:<br><br><i>https://localhost:52810/7.0/CoveoSearchService?wsdl</i><br><br>4 -\\u00a0<span style=\\\"\\\">On your CESMasterServer, this command indicates that port 52810 is open<br><br>telnet localhost 52810</span>\"], \"kav_attribute.url\": [\"/services/data/v34.0/sobjects/KB_Article__kav/ka132000000LJxyAAG\"], \"kav_articlenumber__stripped\": [\"1053\"], \"sfversionnumber\": [3], \"kav_attribute.type\": [\"KB_Article__kav\"], \"kav_validationstatus\": [\"Reviewed\"], \"coveo_foldingparent\": [\"d607de66575964e1dd0a57a1f08ee7cea1bf9b45ba146db613f4466f4e54\"], \"collection\": [\"default\"], \"sfsourceid\": [\"5003200000vJYSiAAO\"], \"date\": [1517572013], \"kav_lastmodifiedby.id\": [\"00560000003q7JgAAI\"], \"kav_datacategoryselections.datacategorygroupname\": [\"Product;Article_Type\"], \"conversionstate\": [0], \"sfvalidationstatus\": [\"Reviewed\"], \"kav_isdeleted\": [false], \"sflastmodifiedbyid\": [\"00560000003q7JgAAI\"], \"kav_lastmodifiedby.name\": [\"Frederic Cote\"], \"kav_articlenumber\": [\"000001053\"], \"permanentid\": [\"c0ab3e69ff69d2d0ca512c74d530b69dcc58372df93595a71ae55e089664\"]}",
      "sfbuildc" : "All",
      "sflastmodifiedbyid" : "00560000003q7JgAAI",
      "sfproductc" : "Coveo Enterprise Search",
      "syssfownerid" : "00560000003q7JgAAI",
      "sourcetype" : "Salesforce",
      "sysconnectortype" : "Salesforce2",
      "sftitle" : "503 An error occurred while making the HTTP request to https://yourCESMasterServer:52810/7.0/CoveoSearchService",
      "rowid" : 5492835,
      "syssfcreatedbyid" : "00560000003q7JgAAI",
      "sfdescriptionc" : "<b>Symptoms</b><br><br>1 - On you Coveo for Sitecore diagnostic page you see:<br><br><i>503 An error occurred while making the HTTP request to https://yourCESMasterServer:52810/7.0/CoveoSearchService. This could be due to the fact that the server certificate is not configured properly with HTTP.SYS in the HTTPS case. This could also be caused by a mismatch of the security binding between the client and the server</i><br><br>2 - When calling restApiServer:8080/rest/search [Default port is 8080, can be changed in config.yml]<br><br>&quot;message&quot; : &quot;Server is unavailable: (https://<span style=\"\">yourCESMasterServer</span>:52810/7.0/CoveoSearchService,java.net.ConnectException: https://yourCESMasterServer:52810/7.0/CoveoSearchService)&quot;<br><br>3 - On your CESMasterServer, you get an &quot;Timeout&quot; message if you&#39;re trying to connect to:<br><br><i>https://localhost:52810/7.0/CoveoSearchService?wsdl</i><br><br>4 - <span style=\"\">On your CESMasterServer, this command indicates that port 52810 is open<br><br>telnet localhost 52810</span>",
      "size" : 6119,
      "sfislatestversion" : "true",
      "sysdocumenttype" : "article",
      "sfcreateddate" : 1481913836000,
      "sfsourceid" : "5003200000vJYSiAAO",
      "clickableuri" : "https://na61.salesforce.com/kA1320000000428",
      "syssource" : "Salesforce Public Items",
      "sfownerid" : "00560000003q7JgAAI",
      "orderingid" : 212384990660371638,
      "sfpublishstatus" : "Online",
      "syssize" : 6119,
      "sysdate" : 1517572013000,
      "sflanguage" : "en_US",
      "sysparents" : "<?xml version=\"1.0\" encoding=\"utf-16\"?><parents><parent name=\"Organization\" uri=\"https://na61.salesforce.com/home/home.jsp\" /><parent name=\"KB_Article__ka\" uri=\"http://www.salesforce.com/org:organization/articletype:KB_Article\" /><parent name=\"503 An error occurred while making the HTTP request to https:__yourCESMasterServer:52810_7.0_CoveoSearchService\" uri=\"https://na61.salesforce.com/kA1320000000428\" /></parents>",
      "sfdcexpandedarticletype" : "All;All|Problem/Fix",
      "sfcurrencyisocode" : "CAD",
      "sfkbid" : "kA1320000000428CAA",
      "sfcausec" : "If the port is open, then it means that the port 52810 was attributed to another service.<br><br>Port 52810 is essential to the Coveo infrastructure.<br><br>To validate this assumption, open a command line Window as an Administrator and run:<br><br>netstat -baon &gt; output.txt<br><br>This will generate a output.txt file in the current repository of your command line window.<br><br>Inside that file, search for 52810. You will find a line like this:<br><br>TCP 0.0.0.0:52810 0.0.0.0:0 LISTENING 1234 <br>[aService.exe]<br><br>*1234 is the PID of aService.exe<br> ",
      "sfdcexpandedproduct" : "All;All|Sitecore",
      "syssfcreateddate" : 1481913836000,
      "sflastmodifieddate" : 1481913895000,
      "sfisvisibleinpkb" : "true",
      "source" : "Salesforce Public Items",
      "sfarticletypec" : "Error Message[Deprecated]",
      "sfisdeleted" : "false",
      "sfisvisibleincsp" : "true",
      "sfvalidationstatus" : "Reviewed",
      "sflastmodifiedbyname" : "Frederic Cote",
      "collection" : "default",
      "sfdcproduct" : [ "Sitecore" ],
      "syssourcetype" : "Salesforce",
      "sfarticletype" : "KB_Article__kav",
      "indexeddate" : 1518187568000,
      "connectortype" : "Salesforce2",
      "filetype" : "SalesforceItem",
      "sfcreatedbyname" : "Frederic Cote",
      "sysclickableuri" : "https://na61.salesforce.com/kA1320000000428",
      "sysfiletype" : "SalesforceItem",
      "language" : [ "English" ],
      "sffirstpublisheddate" : 1453322867000,
      "sfownername" : "Frederic Cote",
      "sitename" : "Community",
      "sysrowid" : 5492835,
      "uri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA1320000000428CAA/language:en_US",
      "syscollection" : "default"
    },
    "Title" : "503 An error occurred while making the HTTP request to https:__yourCESMasterServer:52810_7.0_CoveoSearchService",
    "Uri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA1320000000428CAA/language:en_US",
    "PrintableUri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA1320000000428CAA/language:en_US",
    "ClickUri" : "https://na61.salesforce.com/kA1320000000428",
    "UniqueId" : "42.13819$http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA1320000000428CAA/language:en_US",
    "Excerpt" : "This could be due to the fact that the server certificate is not configured properly with HTTP.SYS in the HTTPS case. ... This could also be caused by a mismatch of the security binding between the...",
    "FirstSentences" : null
  }